subgraph/schema.graphql

```diff
@@ -144,6 +144,19 @@ type Owner @entity {
   tokens: [Token!] @derivedFrom(field: "owner")
 }
 
+type Collection @entity {
+  id: Bytes! #address
+  deployer: Bytes! #address
+  transactionHash: Bytes! #transaction hash
+  owners: [CollectionOwner!]
+}
+
+type CollectionOwner @entity {
+  id: Bytes! # address
+  accessGrantedBy: Bytes! #address
+  transactionHash: Bytes!
+}
+
 type Controller @entity {
```

Comment:
Just catching this now but we should make this either `TokenController` or `CollectionController` right?

Reply:
I can change that to TokenController. Didn't see a benefit but it's a possible change.